src/main/jhipster/src/main/webapp/app/shared/ontology/model/variable-filter-request.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @deprecated
+ */
```

Comment:
can you add more info, just in case we stumble upon this in the future, to know which other mechanism or class we should go to, or what we need to finish this deprecation